tests/fixtures/mod.rs

```diff
@@ -0,0 +1,51 @@
+
+
+use std::process::Command;
+use std::process::Stdio;
+use tiny_http::{Server, Response};
+use std::fs::File;
+use std::thread;
+
+const PAGE: &'static str = "tests/fixtures/index.html";
+
+pub fn spawn_local_http_server() {
+    let server = Server::http("0.0.0.0:8000").unwrap();
+    println!("Spawning http server");
+    thread::spawn(move || {
+        for request in server.incoming_requests() {
+            let response = Response::from_file(File::open(PAGE).unwrap());
+            request.respond(response).unwrap();
+        }
+    });
+}
+
+pub fn get_file_count(pattern: &str, dir: &str) -> usize {
+    // Command being run: `ls | grep .mp3 | wc -w`
```

Comment:
@CohenArthur `echo *.mp3 | wc -w`, did not work because wildcards are a shell feature. Had to use this

Reply:
No worries, I realized it was dumb after posting your comment and discussing with @Skallwar